src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -9,99 +9,223 @@
      * @return first element of specified array
      */ // TODO
     public static String getFirstElement(String[] array) {
-        return null;
+        /*
+        Description
+        Given an array of String objects, return the first element of the array.
+         */
+
+        return array[0];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return second element in specified array
      */
     public static String getSecondElement(String[] array) {
-        return null;
+        return array[1];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return last element in specified array
      */ // TODO
     public static String getLastElement(String[] array) {
-        return null;
+        return array[array.length - 1];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return second to last element in specified array
      */ // TODO
     public static String getSecondToLastElement(String[] array) {
-        return null;
+        return array[array.length - 2];
     }
 
+
+
     /**
      * @param array array of String objects
      * @param value value to check array for
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+
```

Comment:
> Complex is better than complicated

```java
return getNumberOfOccurrences(array, value) > 0;
```

Reply:
I just went through it and I see what you're saying. From the "contains" method, I could have called/returned what you wrote:
return getNumberOfOccurrences(array, value) > 0;
because if value "occurs" at least once, it is > 0 (which = true, which means that the array "contains" it). And that way I can reuse existing methods, instead of re-creating them from scratch.